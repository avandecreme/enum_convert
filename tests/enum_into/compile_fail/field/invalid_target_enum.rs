use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    Struct {
        // Should be #[enum_into(Target::Struct.a)]
        #[enum_into(NonExistent::Struct.a)]
        x: i32,
    },
}

enum Target {
    Struct { a: i32 },
}

fn main() {}
