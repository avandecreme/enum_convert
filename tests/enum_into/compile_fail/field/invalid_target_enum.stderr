error: Field mapping for unknown enum
 --> tests/enum_into/compile_fail/field/invalid_target_enum.rs:8:21
  |
8 |         #[enum_into(NonExistent::Struct.a)]
  |                     ^^^^^^^^^^^
